SpatialGDK/Source/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -51,7 +51,7 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 	return bHasSpatialNetDriver;
 }
 
-bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
+bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(bool bIsPIE, FString& Error)
```

Comment:
How about instead of `bool bIsPIE` we use the `#if WITH_EDITOR` macro?

Reply:
Resolving this one since this comment appears to be duplicated